Sources/SpeziOnboarding/OnboardingConstraint.swift

```diff
@@ -15,6 +15,8 @@ import Spezi
 public protocol OnboardingConstraint: Standard {
     /// Adds a new exported consent form represented as `PDFDocument` to the `Standard` conforming to ``OnboardingConstraint``.
     /// 
-    /// - Parameter consent: The exported consent form represented as `PDFDocument` that should be added.
-    func store(consent: PDFDocument) async
+    /// - Parameters:
+    ///     - consent: The exported consent form represented as `PDFDocument` that should be added.
+    ///     - identifier: A 'String' identifying the consent form as specified in OnboardingConsentView.                
+    func store(consent: PDFDocument, identifier: String) async
```

Comment:
If we change the signature here, I would suggest to make this function throwing and therefore use this as a chance to surface errors to the user and make sure that the user can't progress with the onboarding if the uploading didn't work as expected.